docs/docs/07-forms.md

```diff
@@ -53,12 +53,11 @@ This will render an input that always has a value of `Hello!`. Any user input wi
     this.setState({value: event.target.value});
   },
   render: function() {
-    var value = this.state.value;
-    return <input type="text" value={value} onChange={this.handleChange} />;
```

Comment:
I agree with inlining `this.state.value` because the relationship between the virtual dom and state becomes a tiny bit clearer, but I think you should make it like this while you're at it; to put value and onChange lined up.

```js
return (
  <input
    type="text"
    value={this.state.value}
    onChange={this.handleChange} />
);
```